CHANGELOG.md

```diff
@@ -1,71 +1,120 @@
 # Mattermost Desktop Application Changelog
 
-## UNDER DEVELOPMENT
+## Release v4.0.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost Desktop changelog appears
-in the product's `master` branch to note key changes committed to `master`
-and are on their way to the next stable release.
-When a stable release is pushed, "UNDER DEVELOPMENT" heading is removed
-from the final changelog of the release.
+Release date: December, 2017
```

Comment:
This should be changed to "January, 2018", shouldn't it?

Reply:
Good catch, updated the date